Makefile

```diff
@@ -13,7 +13,8 @@ install:
 	@make .installed
 
 .installed: Pipfile Pipfile.lock
-	@echo "Pipfile(.lock) is newer than .installed, (re)installing"
+	@echo "Pipfile(.lock) is newer than .installed, (re)installing in 3 seconds..."
+	@sleep 3
```

Comment:
I know you really like to have this on EBN, but everywhere else, I cringe when I have to wait 3 seconds. Can we compromise on 1 second delay?

Reply:
Sure. Since it doesn't have as much dependency as EBN, I think it's not that critical here.

Reply:
Then let's leave it out. I prefer not having it, if there are not strong opinions to keep it.

Reply:
Done